pages/code-wrapper.css

```diff
@@ -3,12 +3,23 @@
     border: 2px solid var(--sky-blue);
     border-radius: 1rem;
     margin-bottom: 3rem;
-   
+    max-width: 100%;
+    height: auto;
+}
+
+.code{
+    overflow-y: scroll;
 }
 
 .code-wrapper .hljs{
     background-color: transparent;
     overflow-x: auto;
+    overflow-y: scroll;
+    width: 90%;
+    background: var(--pink8);
+    margin:2rem auto 2rem;
```

Comment:
2rem auto is enough

Reply:
auto is for horizontal only here and the 2rem is for up and bottom as far as i remember,why i gave it like this.